lstmcpipe/config/pipeline_config.py

```diff
@@ -112,9 +111,6 @@ def config_valid(loaded_config):
             f"Please select an allowed `workflow_kind`: {allowed_workflows}"
         )
 
-    prod_type = loaded_config["prod_type"]
-    if prod_type not in allowed_prods:
```

Comment:
Why shouldn't we check the prod type ?
Same comment. 
If we are not checking which type of prod the config file belongs to, erase the `allowed_prods` list above.

Reply:
why would we need to check the `prod_type`???

Reply:
currently `prod_type: prod5`
why would the stages need to know that?

Reply:
the goal here is to remove the hard-coded parts, not replace them

Reply:
No need to check an allowed list. `prod_type` is just additional info but there is nothing mandatory.